File: Modules/Common/Core/SIE.Web.Core/Prints/Commands/InitDefaultTplCommand.cs
using SIE.Core.Prints;
using SIE.Domain;
using SIE.Web.Command;
using System;

namespace SIE.Web.Core.Prints.Commands
{
    /// <summary>
    /// PDA标签补打
    /// </summary>
    public class InitDefaultTplCommand : ViewCommand
    {
        /// <summary>
        /// 命令执行方法
        /// </summary>
        /// <param name="args">视图参数</param>
        /// <param name="scope">作用域</param>
        /// <returns>执行结果</returns>
        protected override object Excute(ViewArgs args, string scope)
        {
            //var logId = Convert.ToDouble(args.Data);
            RT.Service.Resolve<PrinterSettingController>().InitDefaultTpl();

            return true;
        }
    }
}

File: Modules/Common/Core/SIE.Web.Core/Prints/Commands/InitDefaultTplCommand.js
SIE.defineCommand('SIE.Web.Core.Prints.Commands.InitDefaultTplCommand', {
    extend: 'SIE.cmd.Command',
    meta: { text: "初始化默认模板", group: "edit", iconCls: "icon-PrintData icon-blue" },
    execute: function (view, source) {
        //var entity = view && view.getCurrent() && view.getCurrent().data;
        //if (entity == null) return;
        //var id = entity.Id;
        //var dataKey = entity.DataKey;
        var command = Ext.getClassName(this);
        view.execute({
            command: command,
            data: id,
            success: function (res) {
                view.reloadData();
                SIE.Msg.showInstantMessage('初始化完成!'.t(), "初始化", 3);

            }
        });
    }
});